pygluu/containerlib/document/rclone.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import os
+
+from ..utils import exec_cmd
+
+logger = logging.getLogger(__name__)
+
+
+class RClone:
+    def __init__(self, url, username, password):
+        self.url = "localhost:8080/repository/default"
```

Comment:
Please don't hardcode the base URL (`localhost:8080`). It should use the `url` variable